components/status/StatusEmbeddedMedia.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+import type { mastodon } from 'masto'
+
+const { status } = defineProps<{
+  status: mastodon.v1.Status
+}>()
+
+const sanitizedHtml = ref(await sanitizeEmbeddedIframe(status.card?.html || ''))
```

Comment:
`sanitizeEmbeddedIframe` isn't an async function, no? The `await` isn't needed here.
It should be a `computed` from the status passed as a prop (see other derived variables in the other status cards components.

Reply:
ahhh I'll try :) always still not sure which one to use. My understanding is `computed()` when combining variables from `ref()`?

Reply:
Pushed a change, not sure if it is the best... I use `computed` from the `status` prop, but can't make it work without `ref` and `await` when using `sanitizeEmbeddedIframe`, because that function returns a `Promise<string>` 😬 -- any ideas?

Here's what I ended up with:
```ts
const cardHtml = $computed(() => status.card?.html ?? '')
const sanitizedHtml = ref(await sanitizeEmbeddedIframe(cardHtml))
```

Reply:
Ah, found how we do it for content in StatusBody... will do that too

Reply:
Done!